.github/workflows/test.yml

```diff
@@ -219,7 +219,7 @@ jobs:
       - name: test liveness
         if: env.GIT_DIFF
         run: |
-          ./contrib/localnet_liveness.sh 100 5 50 localhost
+          ./contrib/localnet_liveness.sh 100 5 50 127.0.0.1
```

Comment:
The liveness test isn't passing: https://github.com/cosmos/cosmos-sdk/actions/runs/5920973911/job/16052925729?pr=17470

Reply:
Yea but I can't test docker related stuff, will try after IT stop hurting my laptop

Reply:
liveness test need pass `listen-ip-address` as `0.0.0.0` since docker can't expose port on `127.0.0.1`